DuckDuckGo/ContentBlockerViewController.swift

```diff
@@ -27,8 +27,8 @@ class ContentBlockerViewController: UITableViewController {
     @IBOutlet weak var httpsLabel: UILabel!
     @IBOutlet weak var blockCountCircle: UIImageView!
     @IBOutlet weak var blockCount: UILabel!
-    @IBOutlet weak var blockingEnabledToggle: UISwitch!
     @IBOutlet weak var blockThisDomainToggle: UISwitch!
+    @IBOutlet weak var tutorialText: UILabel!
```

Comment:
And repetition in the UI is the worst, would rather have a few more lines of code, tbh.  Good call with the UITableViewCell, that makes sense.